go/vt/vttablet/tabletserver/health_streamer.go

```diff
@@ -380,7 +382,13 @@ func getChangedTableNames(ctx context.Context, conn *connpool.DBConn) ([]string,
 	}
 	alloc := func() *sqltypes.Result { return &sqltypes.Result{} }
 	bufferSize := 1000
-	err := conn.Stream(ctx, mysql.DetectSchemaChange, callback, alloc, bufferSize, 0)
+
+	schemaChangeQuery := mysql.DetectSchemaChange
+	// If views are enabled, then views are tracked/handled separately and schema change does not need to track them.
+	if hs.viewsEnabled {
+		schemaChangeQuery = mysql.DetectSchemaChangeOnlyBaseTable
+	}
```

Comment:
Can't we always use `mysql.DetectSchemaChangeOnlyBaseTable`?

Reply:
There is a possibility of users creating views on MySQL and providing that in vschema as table, so schema tracking should be able to populate column list for those when vitess managed views are not enabled.